### DISCUSSION TITLE
Multiple admin panel depending on user role

### DISCUSSION THREAD

**Discussion by InSecker** [strapi/strapi#8506] [Legacy Topics]
Hi there!

I wonder if there is a way to redirect logged user depending on its role to a specific endpoint.

An admin is redirected to the default Strapi admin panel.
An editor is redirected to another custom page, not dependant of Strapi BO.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
